test/koa-mysql/app/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash

set -eu
```

Make sure it fails when a child process fails.